src/pretalx/orga/templates/orga/review/base.html

```diff
@@ -0,0 +1,45 @@
+{% extends "orga/base.html" %}
+{% load i18n %}
+{% block headline %}
+    {% if submission %}
+        {% blocktrans with title=submission.title %}
+            Reviews for submission »{{ title }}«
+        {% endblocktrans %}
+        ({{ submission.display_speaker_names }})
+        <a href="{% url "orga:submissions.content.view" event=request.event.slug pk=submission.pk %}">
```

Comment:
Using `pk=` here causes a `NoReverseMatch` error; `code=` makes it work, but that's kind of backwards, as the pk is explicitly not the code. `SubmissionViewMixin` also seems like the code should be given, not the pk. Welp.

Reply:
Yeah, parts of this PR are from the olden times™ when URLs were different. I've added a commit that uses urlman-urls instead.